src/requestManager.cpp

```diff
@@ -297,15 +297,17 @@ void CUnknownObj::AddSource(CNode* from)
       }
       CNodeRequestData req(from);
       for (ObjectSourceList::iterator i = availableFrom.begin(); i != availableFrom.end(); ++i)
-        {
-	  if (i->desirability < req.desirability)
-	    {
+      {
+          if (i->desirability < req.desirability)
+          {
               availableFrom.insert(i, req);
-              return;
-	    }
-        }
+              return true;
+           }
+      }
       availableFrom.push_back(req);
+      return true;
     }
```

Comment:
I know this part is mostly formatting and updating return type to fit method but this raw loop should be refactored into a `find_if` or using some std algorithm.